src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
+i.icon.universal.access:before { content: "\f29a"; }
+i.icon.alternative.wheelchair:before { content: "\f29b"; }
+i.icon.blind:before { content: "\f29d"; }
+i.icon.audio.description:before { content: "\f29e"; }
+i.icon.volume.control.phone:before { content: "\f2a0"; }
+i.icon.braille:before { content: "\f2a1"; }
+i.icon.assistive.listening.systems:before { content: "\f2a2"; }
+i.icon.american.sign.language.interpreting:before { content: "\f2a3"; }
+i.icon.deafness:before { content: "\f2a4"; }
```

Comment:
New icon: `deafness`, an icon representing complete loss of hearing. Depicts an ear and a line with a line through it, showing no sounds passing through. _I propose to be this the main class for this icon, being aliases_ `deaf` _and_ `hard of hearing`.